assets/js/state/lastExecutions.js

```diff
@@ -42,11 +44,36 @@ export const lastExecutionsSlice = createSlice({
 
       const lastExecutionState = {
         ...initialExecutionState,
+        executionStarted: true,
         error,
       };
 
       state[groupID] = lastExecutionState;
     },
+    setExecutionStarted: (state, { payload }) => {
```

Comment:
I think that the next would be a better option:
- Change the current `setExecutionRequested` function to simply put ` requested` value in the status field
- Change this `setExecutionStarted` to do what the current `setExecutionRequested` does, which was working fine, except the data was not coming from wanda

Reply:
+1 using the status and transforming it to a tristate,

Reply:
done